src/praisonai/praisonai/tools/skill_manage.py

```
@@ -0,0 +1,428 @@
+            
+            pending_path.rename(active_path)
+            
+            # Clean up proposal metadata
+            proposal_file = active_path / ".proposal.json"
+            if proposal_file.exists():
+                proposal_file.unlink()
+            
+            self._log_action("approve", name, False)
+            return f"✅ Skill '{name}' approved and activated."
+            
+        except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)